hardhat.config.ts

```diff
@@ -95,4 +96,11 @@ task('allowed-tokens', 'Retrieves a list of allowed tokens')
   }
 );
 
+task('remove-tokens', 'Removes a list of tokens')
+  .addPositionalParam('tokenList', 'list of tokens')
```

Comment:
The parameter name should be exactly the same as the one used, otherwise it won't be parsed.
```suggestion
  .addPositionalParam('tokenlist', 'list of tokens')
```

Reply:
Fixed, thanks.